src/handlers/dashboard.rs

```diff
@@ -9,18 +11,14 @@ use axum::{
     Extension,
 };
 
-use crate::common::repository::DbReadAll;
```

Comment:
Why delete this line? We use the ReadAll in game_repository.read_all() - for that I think we need that imported here.

Reply:
It has simply moved to line 3.